### ISSUE TITLE
Word highlighter error when opening accessibility help

### ISSUE THREAD

**Issue by amunger** [microsoft/vscode#229224]
1. open a text editor
2. open the accessibility help alt+f2

:bug: error in window log (no noticeable functionality loss)

2024-09-20 11:57:30.396 [error] Unable to resolve resource accessible-view:accessible-view-editor
...
    at ResourceModelCollection.doCreateReferencedObject (vscode-file://vscode-app/c:/src/vscode/out/vs/workbench/services/textmodelResolver/common/textModelResolverService.js:84:15)
    at ResourceModelCollection.doCreateReferencedObject (vscode-file://vscode-app/c:/src/vscode/out/vs/workbench/services/textmodelResolver/common/textModelResolverService.js:82:25)
    at async AsyncReferenceCollection.acquire (vscode-file://vscode-app/c:/src/vscode/out/vs/base/common/lifecycle.js:531:28)
    at async TextModelResolverService.createModelReference (vscode-file://vscode-app/c:/src/vscode/out/vs/workbench/services/textmodelResolver/common/textModelResolverService.js:205:16)
    at async WordHighlighter._run (vscode-file://vscode-app/c:/src/vscode/out/vs/editor/contrib/wordHighlighter/browser/wordHighlighter.js:550:35)

**Comment by Yoyokrazy** [comment]
no longer can repro. likely fixed with the accessibility candidate a month or two ago that megan found.
